rpc/core/env.go

```diff
@@ -70,9 +76,14 @@ type Environment struct {
 	StateStore     sm.Store
 	BlockStore     sm.BlockStore
 	EvidencePool   sm.EvidencePool
-	ConsensusState Consensus
-	P2PPeers       peers
-	P2PTransport   transport
+	ConsensusState consensusState
+
+	// Legacy p2p stack
+	P2PPeers     peers
+	P2PTransport transport
```

Comment:
These methods appear to be implemented _on the node_. Are they strictly related to the legacy p2p stack? if not, should we rename to something not directly p2p related?